docs/runtime/nodejs-metrics.md

```diff
@@ -0,0 +1,310 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Node.js Runtime
+--->
+
+# Semantic Conventions for Node.js Runtime Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for Node.js Runtime metrics in OpenTelemetry.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Experimental](#experimental)
+  - [Metric: `nodejs.eventloop.delay.min`](#metric-nodejseventloopdelaymin)
+  - [Metric: `nodejs.eventloop.delay.max`](#metric-nodejseventloopdelaymax)
+  - [Metric: `nodejs.eventloop.delay.mean`](#metric-nodejseventloopdelaymean)
+  - [Metric: `nodejs.eventloop.delay.stddev`](#metric-nodejseventloopdelaystddev)
+  - [Metric: `nodejs.eventloop.delay.p50`](#metric-nodejseventloopdelayp50)
+  - [Metric: `nodejs.eventloop.delay.p90`](#metric-nodejseventloopdelayp90)
+  - [Metric: `nodejs.eventloop.delay.p99`](#metric-nodejseventloopdelayp99)
```

Comment:
This has been answered in previous comments, but I'd like to reiterate: it would be really nice to unify all this into one single histogram.

This is not blocking for merging those metrics as experimental, but I think should be thoroughly evaluated before declaring those metrics as stable. I wonder if there's any possibility for the runtime to expose the raw event loop delay and to feed this into a native OTel histogram.

Reply:
Agree that this is the ideal scenario, so until then I will keep those separate and if we find a out to convert, I will update the semantic here.